### PR TITLE
FEATURE: QA-19130 Migrate G2C DateFilteringOnInsightTest

### PR DESCRIPTION
**Pull request by duclequang**
---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |
| **Compatibility matrix test commands - TIGER Backend**  |                                                            |
| `extended test - matrix-test <AIO_version>`             | Run integrated tests against AIO versions.                 |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

`<AIO_version>` in commands is used to start test with multiple AIO instances - each instance in triggered by one jenkins build

-   To run with `master` and `stable`, execute command `extended test - matrix-test master,stable` or `extended test - matrix-test latest`
-   To run with specific version,ex: `2.3.0` and `2.3.1`, execute command `extended test - matrix-test 2.3.0,2.3.1`
-   In case `<AIO_version>` is empty, read versions from file `compTigerVersions.txt` of this repo

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15549/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17059/ : FAILURE in 8m 03s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15573/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17082/ : SUCCESS in 7m 14s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15574/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17083/ : SUCCESS in 7m 13s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/782/  
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/783/

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15595/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17110/ : SUCCESS in 7m 29s

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 1]
```diff
@@ -79,3 +84,106 @@ describe("Insights on dashboard", () => {
...
 });
+
+describe("Date filtering on Insight", () => {
+    beforeEach(() => {
+        Navigation.visit("dashboard/dashboard-date-filtering-on-insight-scenario");
+        editMode.isInEditMode(false).edit().isInEditMode();
+    });
+
+    it(
+        "Remember Last Setting After Selecting Another Insight",
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:95]: lower case, check other places

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 2]
```diff
@@ -79,3 +84,106 @@ describe("Insights on dashboard", () => {
...
+        { tags: ["checklist_integrated_tiger"] },
+        () => {
+            new Widget(0).waitChartLoaded();
+
+            const widgetConfiguration = new WidgetConfiguration(0).open().openConfiguration();
+            widgetConfiguration.selectDateDataset(DATASET_CREATED);
+
+            new Widget(1).hasTitle("Column with two measures by date");
+
+            widgetConfiguration.hasDatasetSelected(DATASET_CREATED);
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:105]: check again in Graphene test, does it check the setting of the first widget after select the second one, or does it check the setting of the second widget to make sure it has the same setting as the first widget?

Your test seems not meet any of above 2 cases:
Your test flow:
1. select widget0 -> open configuration --> select Dataset_created
2. check widget1 title 
3. verify again widget0 date configuration = created

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 3]
```diff
@@ -79,3 +84,106 @@ describe("Insights on dashboard", () => {
...
+        widgetConfiguration.isDateFilterExist(true);
+        widgetConfiguration.disableDateFilter();
+        widgetConfiguration.isDateFilterExist(false);
+    });
+
+    it("Enable Date Filter", { tags: ["checklist_integrated_tiger"] }, () => {
+        new Widget(0).waitChartLoaded();
+        const widgetConfiguration = new WidgetConfiguration(0).open().openConfiguration();
+
+        widgetConfiguration.enableDateFilter();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:122]: missing check date filter is disable first

File: libs/sdk-ui-tests-e2e/reference_workspace/fixtures/goodsales/tiger_metadata_extension.json [hunk 4]
```diff
@@ -1761,6 +1761,108 @@
                 "description": "",
                 "id": "5c8244eb-6c86-488c-9df1-37d46e5450ed",
                 "title": "KD with table has many rows/columns"
+            },
+            {
+                "content": {
+                    "filterContextRef": {
```
[the remark this answers is not in the document]
Comment (duclequang) [reply libs/sdk-ui-tests-e2e/reference_workspace/fixtures/goodsales/tiger_metadata_extension.json:1767]: This snippet code generate by "rushx create-ref-workspace"

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15601/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17116/ : FAILURE in 8m 39s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15602/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17117/ : FAILURE in 7m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15603/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17118/ : FAILURE in 7m 06s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/805/ 
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/804/

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15604/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17119/ : FAILURE in 7m 08s

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 5]
```diff
@@ -79,3 +87,95 @@ describe("Insights on dashboard", () => {
...
+            .getDataLabelValues()
+            .should("deep.equal", ["$4,108,360.80", "$2,267,528.48", "$3,461,373.87"]);
+    });
+
+    it("remember disabled date filter after saving", { tags: ["checklist_integrated_tiger"] }, () => {
+        secondWidget.waitChartLoaded();
+        secondWidgetConfiguration.open().openConfiguration()
+            .disableDateFilter().isDateFilterExist(false);
+
+        dateFilter.openAndSelectDateFilterByName(DateFilterValue.THIS_MONTH).apply();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:144]: why do you change date filter here?

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 6]
```diff
@@ -79,3 +87,95 @@ describe("Insights on dashboard", () => {
...
+        editMode.save().isInEditMode(false).edit().isInEditMode();
+
+        secondWidgetConfiguration.open().openConfiguration().isDateFilterExist(false);
+    });
+
+    it("remember enabled date filter after saving", { tags: ["checklist_integrated_tiger"] }, () => {
+        secondWidget.waitChartLoaded();
+        secondWidgetConfiguration.open().openConfiguration()
+            .enableDateFilter().selectDateDataset(DATASET_SNAPSHOT).isDateFilterExist();
+        dateFilter.openAndSelectDateFilterByName(DateFilterValue.ALL_TIME).apply();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:156]: why do you change date filter here?

File: libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts [hunk 7]
```diff
@@ -113,6 +113,30 @@ export class WidgetConfiguration {
...
+    disableDateFilter() {
+        cy.get("body").then((body) => {
+            if (body.find(".s-filter-date-dropdown").length > 0) {
+                cy.get(".s-date-filter-checkbox").click();
+            }
+        });
+        return this;
+    }
+
+    isDateFilterExist(exist = true) {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts:135]: confusing name, should name as isDateDataSetDropdownVisible

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts [hunk 8]
```diff
@@ -79,3 +87,95 @@ describe("Insights on dashboard", () => {
...
+            .isDateFilterExist();
+    });
+
+    it("remember modified date filter after saving", { tags: ["checklist_integrated_tiger"] }, () => {
+        widget.waitChartLoaded();
+
+        widgetConfiguration.open().openConfiguration()
+            .selectDateDataset(DATASET_CREATED).hasDatasetSelected(DATASET_CREATED);
+
+        dateFilter.openAndSelectDateFilterByName(DateFilterValue.THIS_MONTH).apply();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:174]: why do you change date filter here?

File: libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts [hunk 9]
```diff
@@ -113,6 +113,30 @@ export class WidgetConfiguration {
...
+                cy.get(".s-date-filter-checkbox").click();
+            }
+        });
+        return this;
+    }
+
+    disableDateFilter() {
+        cy.get("body").then((body) => {
+            if (body.find(".s-filter-date-dropdown").length > 0) {
+                cy.get(".s-date-filter-checkbox").click();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts:129]: use toggleDateFilter()

File: libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts [hunk 10]
```diff
@@ -113,6 +113,30 @@ export class WidgetConfiguration {
         return this;
     }
 
+    enableDateFilter() {
+        cy.get("body").then((body) => {
+            cy.log("length.  : " + body.find(".s-filter-date-dropdown").length);
+            if (body.find(".s-filter-date-dropdown").length == 0) {
+                cy.get(".s-date-filter-checkbox").click();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts:120]: use toggleDateFilter()

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 5 above)
Comment (duclequang) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:144]: add more action to editmode can save

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 6 above)
Comment (duclequang) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:156]: add more action to editmode can save

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 8 above)
Comment (duclequang) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:174]: add more action to editmode can save

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15609/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17124/ : FAILURE in 7m 31s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15610/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17125/ : FAILURE in 7m 09s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15611/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17126/ : SUCCESS in 7m 08s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/827/ 
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/826/

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 5 above)
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:153]: in bear, we don't need to add any action to enable save button
toggle date filter checkbox is an action, and save button should be enable if we toggle it
confirm again, is it a platform issue?

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 8 above)
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:190]: no need

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 6 above)
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:169]: no need

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts (hunk 2 above)
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts:108]: how can it check date dimension of widget 1 while configuration of widget2 is opening?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15612/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17131/ : SUCCESS in 7m 12s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/829/ 
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/828/

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15614/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17132/ : SUCCESS in 7m 12s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15624/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17142/ : SUCCESS in 8m 50s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15627/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17145/ : FAILURE in 8m 16s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15629/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17148/ : FAILURE in 8m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15630/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17150/ : FAILURE in 8m 29s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/837/ 
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/836/

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15631/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17151/ : FAILURE in 7m 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15632/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17152/ : SUCCESS in 7m 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15635/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17156/ : SUCCESS in 7m 14s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15641/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17162/ : SUCCESS in 7m 13s

**Comment by duclequang** [comment]
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/859/ 
https://checklist.intgdc.com/job/cypress-tests/job/gooddata-ui-sdk-cypress-checklist-worker/858/

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15647/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17168/ : SUCCESS in 7m 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/15648/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17171/ : SUCCESS in 7m 37s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4060

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/3186/ : SUCCESS in 9m 26s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/17172/ : SUCCESS in 7m 20s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/785/ : SUCCESS in 10m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4345/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1066/ : FAILURE in 1m 51s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/709/ : SUCCESS in 9m 21s

**Pull request merged**